core/controlplane/config/templates/cloud-config-controller

```diff
@@ -5167,6 +5167,9 @@ write_files:
         name: kube-system
         annotations:
           iam.amazonaws.com/permitted: ".*"
+        labels:
+          name: kube-system
```

Comment:
Makes sense! But it would be even better if we could find a more generic way. Perhaps `kubeSystemNamespaceLabels:` in cluster.yaml?( So verbose!

WDYT?

Reply:
@mumoshu :point_up:

Reply:
Sorry for the delay! I'd prefer a hash containing the whole labels i.e. `kubeSystemNamespaceLabels`.

I'm not yet sure if everyone could agrees on this specific label `name: kube-system`.
I also considered something like `kube-aws/name: kube-system` that looks more specific to kube-aws. But it wouldn't be optimal. There should be one or more namespaces that you would like to add labels as well, and adding `kube-aws/name: foo` to every ns managed outside of kube-aws seems wrong.

Something like `kubeSystemNamespaceLabels` seems to provide enough options for users to consistently label namespaces across the cluster, regardless of it is managed by kube-aws or not.

WDYT?

Reply:
Makes complete sense.  thanks for getting back to me.  I will adjust and resubmit.